sktm/reporter.py

```diff
@@ -28,6 +28,7 @@
 SUMMARY_MERGE_FAILURE = 1
 SUMMARY_BUILD_FAILURE = 2
 SUMMARY_TEST_FAILURE = 3
+SUMMARY_CALL_TRACE = 4
```

Comment:
This name is somewhat confusing and doesn't match the pattern so far. Could you call it something like `SUMMARY_CALL_TRACE_FOUND`, or `SUMMARY_TRACE_FOUND`, or `SUMMARY_TRACE_WARNING`?